### ISSUE TITLE
NTR: Indican increased in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8823]
New term request Indican increased in urine (segal_080313143816): Indoxyl sulfate; indicanuria

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000130
- label: Elevated urinary indican level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:6417635
- def: Detection of indican in the urine. Urinary indican concentration may be elevated in small bowel bacterial overgrowth. Indican is produced by bacterial deconjugation of dietary tryptophan to indole which is absorbed at all levels of the intestine, but principally from the small bowel.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8823
- synonyms:
